.gitignore

```diff
@@ -42,26 +38,37 @@ packages/*/*/*.d.ts
 packages/**/*.test-vrt.ts
 *.tsbuildinfo
 
+# ts + css assets
+src/**/*.css.ts
+packages/**/*.css.ts
+styles/**/*.css.ts
+
+# compiled assets
 projects/**/*.js
 projects/**/*.js.map
 projects/**/*.d.ts
 !projects/*/global.d.ts
 
+# build logs
```

Comment:
I wonder what outputs these two... 🤔

Reply:
Do you think they might be outdated?  I could remove them and we can always add back if they reappear.

Reply:
Sounds good to me!